src/GraphQL.Tests/Utilities/SchemaPrinterTests.cs

```diff
@@ -854,6 +854,36 @@ public void prints_input_type_with_default()
             AssertEqual(print(schema), expected);
         }
 
+        [Fact]
+        public void prints_input_type_with_default_null_value()
+        {
+            var root = new ObjectGraphType { Name = "Query" };
+            root.Field<NonNullGraphType<StringGraphType>>(
+                "str",
+                arguments: new QueryArguments(
+                    new QueryArgument<NonNullGraphType<SomeInputType2>> { Name = "argOne", DefaultValue = new SomeInput2 { Names = null } })
```

Comment:
Probably, yes, but `names` is `[String]!` and I suspect that the problem needs to be determined earlier - at the stage of the schema initialization. Surely in the specification there is a necessary requirement. We need to add one more check (for default values) in the `SchemaValidationVisitor`. See `InputObjectGraphType.IsValidDefault` too.

Reply:
I mean that I'm not against adding
```c#
            if (value == null)
                return "null";
```

but this tests shows IMO another problem.

Reply:
Fixed test schema so that `names` is `[String]`.  NRE still exists without the fix in this PR.  This PR solves the NRE.

Reply:
ok